filterpage.html

```diff
@@ -0,0 +1,396 @@
+        height: 20px;
+        width: 20px;
+      }
+
+      .filter-image-headline-wrappper {
+        display: flex;
+        padding-left: 10px;
+      }
+
+      .filter-headline {
+        font-size: 24px;
+        font-weight: 800;
+        color: #3c3b3b;
+        padding-left: 15px;
+        padding-top: 5px;
+      }
+
+      .make-car-list,
+      .body-wrapper {
+        border: 1px solid #e0dede;
+        border-radius: 5px;
+        margin: 10px;
+      }
+
+      .make-car-list,
+      .selection-wrapper,
+      .body-wrapper {
+        padding: 10px;
+      }
+
+      .checkbox-wrapper {
+        padding: 5px;
+        list-style: none;
+        font-family: arial;
+      }
+
+      .make-heading,
+      .body-heading {
+        font-size: 20px;
+        font-weight: 500;
+        color: #302a2a;
+        margin-top: 5px;
+        margin-bottom: 5px;
+        font-family: Arial, Helvetica, sans-serif;
+      }
+
+      .apply-filters__btn {
+        width: calc(100% - 20px);
+        position: absolute;
+        right: 10px;
+        left: 10px;
+        bottom: 5px;
+      }
+
+      .make-and-arrow-wrapper,
+      .body-and-arrow-wrapper {
+        display: flex;
+        justify-content: space-between;
+      }
+
+      .arrow,
+      .body-arrow {
+        padding: 10px 0;
+        height: 20px;
+      }
+
+      .hide {
+        display: none;
+      }
+
+      .make-list-wrapper,
+      .body-list-wrapper {
+        transition: height 300ms;
+        display: flex;
+        flex-wrap: wrap;
+      }
+
+      .collapse {
+        height: 0;
+        overflow: hidden;
+      }
+
+      .downward {
+        transform: rotate(180deg);
+      }
+
+      .selected-result {
+        color: #03b19f;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="open-filter-page-wrapper">
+      <h2 class="filters">Filters</h2>
+      <button class="open-filter__btn">Open Filter</button>
+
+      <div class="selected-filters">Selected Filters:</div>
+
+      <div class="selection-status">No filters selected!</div>
+      <div class="body-selection-status"></div>
+    </div>
+
+    <div class="selection-wrapper hide">
+      <div class="filter-image-headline-wrappper">
+        <button class="cross-symbol-top">
+          <img
+            src="icons8-close-32.png"
+            alt="cross symbol"
+            class="cross-symbol-img"
+          />
+        </button>
+        <h2 class="filter-headline">Filters</h2>
+      </div>
+
+      <div class="make-car-list">
+        <div class="make-and-arrow-wrapper">
+          <h2 class="make-heading">Make</h2>
+          <img
+            src="icons8-more-than-48.png"
+            alt="Upwards arrow"
+            class="arrow"
+          />
+        </div>
+
+        <div class="make-list-wrapper collapse">
+          <div>
+            <ul>
+              <li class="checkbox-wrapper">
+                <input
+                  type="checkbox"
+                  id="Maruti Suzuki"
```

Comment:
The id should be of one word only.
```suggestion
                  id="MarutiSuzuki"
```